source/includes/documentation/_material_ui_elements.md

```diff
@@ -2086,10 +2090,11 @@ Available methods in Java JDI Light:
 
 |Method | Description | Return Type
 | --- | --- | ---
-**is()** | Returns object for work with assertions | ButtonGroupAssert
-**getButtonByIndex(int)** | Get button by index | MaterialButton
-**getButtonByText(String)** | Get button by text | MaterialButton
-**getAllButtons()**        | Get all buttons in a block | Collection\<MaterialButton\>
+**button(int)**        | Get button with index '{0}'| Button
+**button(String)**        | Get button with text '{0}'| Button
+**getAllButtons**        | Get all buttons in '{name}'| List\<Button\>
```

Comment:
1. `getAllButtons()`
2. I'd prefer old description as they are **less technicall** and **telling more info without context** (e.g. what is `{0}` or `{name}`?)